### DISCUSSION TITLE
Unable to connect my core lightning node to RTL

### DISCUSSION THREAD

**Discussion by Daunus** [Ride-The-Lightning/RTL#1091] [Q&A] [answered]
Hi all,

I am running core lightning remotely on a ubuntu machine and am trying to connect it to RTL on a different machine.

I have spent many hours with this already but unfortunately have not been able to do this. I have comp sci background  but am kinda mediocre. 

Unfortunately, I found no tutorials or guides on how to do this. It would be awesome if someone smarter than me made a video tutorial that I could follow step by step.

I have so far:

-  set up core lightning successfully. That was easy.

- Set up lightning rest plug-in for core lightning.  Although it's possible I did not configure it correctly. I have not been able to connect it to ZEUS wallet either. https://github.com/Ride-The-Lightning/c-lightning-REST

- I set up RTL on a separate ubuntu machine. I managed to run it but I get an error. I am almost certain that my config is not set up correctly. The sample config is for LND and I am not sure what it should be for core lightning.

Anyhow when I try to login I get the below error.

Would anyone be kind enough to help me solve the below error?

<img width="1504" alt="Screen Shot 2022-09-06 at 11 54 30 am" src="https://user-images.githubusercontent.com/6956228/188535634-9c00cf2e-71a2-41a2-b805-0e5eac78c796.png">

**Comment by saubyk** [comment, marked as answer]
Hi @Daunus there are three things you need to ensure to connect RTL remotely to your core lightning node via cl-rest:
1. You need to know and specify the local IP address of your ubuntu machine in RTL-Config.json
2. You need to copy the cl-rest `access.macaroon` file from your ubuntu machine over to the machine on which your running RTL and specify it's location in RTL-Config.json
3. The port on which cl-rest is serving on your ubuntu machine should be accessible to other machines on your local network

If you're still not able to make progress, please share your RTL-Config (masking any sensitive info) so that we can troubleshoot further. Hope this helps.

**Reply by Daunus** [reply]
I managed to resolve the issue.

It turns out my config was not really the problem.

I set my core lightning on AWS and a security rule was blocking inbound access. 

Thanks for your response and help.

**Comment by ShahanaFarooqui** [comment]
@Daunus 

- Core lightning's configuration options in RTL are a little different than LND. Please refer to CLN's setup documentation [here](https://github.com/Ride-The-Lightning/RTL/blob/master/.github/docs/Core_lightning_setup.md#prep-for-execution). 

- Fun fact: You can also configure multiple nodes (LND & CLN both) in a single RTL instance.
